juno/assetlist.json

```diff
@@ -1428,6 +1428,26 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/yfd.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/yfd.svg"
       }
+  },
+  {
+    "denom_units": [
+      {
+        "denom": "factory/juno1qly4zcmzr2gyxtze5yt9chv2srczwwunppxjfh/NEXX",
+        "exponent": 0
+      },
+      {
+        "denom": "unexx",
```

Comment:
I'm guessing this denom is supposed to be 'nexx' and not 'unexx'?

Reply:
That's correct, thank you